src/garage/torch/algos/loss_function_utils.py

```diff
@@ -4,7 +4,25 @@
 
 
 def compute_advantages(discount, gae_lambda, max_len, baselines, rewards):
-    """Calculate advantages."""
+    """
+    Calculate advantages.
+
+    Advantages are a discounted cumulative sum.
+
+    The discount cumulative sum can be represented as an IIR
+    filter ob the reversed input vectors, i.e.
+       y[t] - discount*y[t+1] = x[t]
+           or
+       rev(y)[t] - discount*rev(y)[t-1] = rev(x)[t]
+
+    Given the time-domain IIR filter step response, we can
+    calculate the filter response to our signal by convolving the
+    signal with the filter response function. The time-domain IIR
+    step response is calculated below as discount_filter:
+        discount_filter =
+            [1, discount, discount^2, ..., discount^N-1]
+            where the epsiode length is N.
+    """
```

Comment:
this still requires a full docstring.

Reply:
I will put full docstring in "vpg_coverage" branch